README.md

```diff
@@ -53,6 +53,20 @@ defmodule Person do
 end
 ```
 
+Is possible to define the schema with `string: ` param in order to directly use a string to geerate content
+```elixir
+...
+@content """
+id,first_name,last_name,email,gender,ip_address,date_of_birth
+1,Ivory,Overstreet,ioverstreet0@businessweek.com,Female,30.138.91.62,10/22/2018
+2,Ulick,Vasnev,uvasnev1@vkontakte.ru,Male,35.15.164.70,01/19/2018
+3,Chloe,Freemantle,cfreemantle2@parallels.com,Female,133.133.113.255,08/13/2018
+"""
+
+schema string: @content do
```

Comment:
Maybe `content` is better then `string`?

Reply:
Changed with `data`